README.md

```diff
@@ -724,6 +724,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
   * [Flox](https://github.com/devfake/flox) `⚠` - Self hosted movie watch list with a rating based on a 3-point system for good, medium and bad. ([Demo](http://80.240.132.120/flox-demo/public/)) `MIT` `PHP`
   * [GeneWeb](https://geneweb.tuxfamily.org/wiki/GeneWeb) - GeneWeb is an open source genealogy software written in OCaml. It comes with a Web interface and can be used off-line or as a Web service. ([Demo](https://demo.geneweb.tuxfamily.org/gw7/), [Source Code](https://github.com/geneweb/geneweb)) `GPLv2` `OCAML`
   * [King Phisher](https://github.com/securestate/king-phisher) - King Phisher is a tool for testing and promoting user awareness by simulating real world phishing attacks. `BSD` `Python`
+  * [LicAPI](https://projects.miguelpiedrafita.com/LicAPI) - A PHP API to manage license info ([Source Code](https://github.com/m1guelpf/LicAPI), [Demo](https://demo.miguelpiedrafita.com/LicAPI)) `MPLv2` `PHP`
```

Comment:
Source Code and Demo links are the wrong way around. You're also missing a full stop after the description.